dhis-2/dhis-services/dhis-service-program-rule/src/test/java/org/hisp/dhis/programrule/engine/ProgramRuleEngineDescriptionTest.java

```diff
@@ -192,6 +208,15 @@ void testProgramRuleWithTextTrackedEntityAttribute()
         assertTrue( result.isValid() );
     }
 
+    @Test
+    void testProgramRuleWithCalculatedValueRuleVariable()
+    {
+        RuleValidationResult result = validateRuleCondition( conditionWithCalculatedValueRuleVariables, program );
+        assertNotNull( result );
```

Comment:
I think we and I also like grouping tests in the style of arrange, act, assert. So I can easily spot what is being tested. So here I would add a newline before the first assertion.

Reply:
resolved

Reply:
what i normally do is run a speedy-spotless plugin to format the code.

Reply:
It will not group the statements for you, so not insert a newline